Tests/IntegrationTests/Components Tests/Gift Card/GiftCardComponentTests.swift

```diff
@@ -123,6 +123,43 @@ class GiftCardComponentTests: XCTestCase {
         XCTAssertEqual(securityCodeItemTitleLabel?.text, "Security code", "cvc title changes based on payment method")
     }
 
+    func testMealVoucherDetails() {
+
+        // Given
+        let paymentMethod = MealVoucherPaymentMethod(type: .mealVoucherSodexo, name: "Sodexo")
+        sut = GiftCardComponent(
+            partialPaymentMethodType: .mealVoucher(paymentMethod),
+            context: context,
+            amount: amountToPay,
+            publicKeyProvider: publicKeyProvider
+        )
+
+        // When
+        setupRootViewController(sut.viewController)
+        wait(for: .milliseconds(300))
```

Comment:
This should not be needed as the root view controller already does some waiting 🤔

Reply:
Fixed [16e51aa](https://github.com/Adyen/adyen-ios/pull/1904/commits/16e51aa6a1ae07b7a1ffa096db5c980c59598b46)